test/fixtures/alias-basic.json

```diff
@@ -0,0 +1,13 @@
+{
+  "input": {
+    "type": "alias",
+    "previousId": "user-id",
+    "userId": "new-user-id",
+    "timestamp": "2014"
+  },
+  "output": {
+    "auth_token": "OTk1MDRmZWExN2Q5YjcwODA1ZTQ3MGE2NzJhZjFjNWI2MDhlYjg4ZjozODUzNzJlMjMwOWQ2NTg0NTQyNDUwMmM0NzQwN2ZlNDJiM2ZmOWQz",
```

Comment:
some of them had it, some of them didn't. I put it in all of them so that we can match them against what we are actually sending